gen/gen.ml

```diff
@@ -48,9 +49,9 @@ let no_tensor_options =
 let prefixed_functions =
   Set.of_list
     (module String)
-    [ "add"; "add_"; "div"; "div_"; "mul"; "mul_"; "sub"; "sub_"; "nll_loss" ]
+    [ "add"; "add_"; "div"; "div_"; "mul"; "mul_"; "sub"; "sub_"; "nll_loss"; "to_mkldnn" ]
 
-let excluded_prefixes = [ "_thnn_"; "_th_"; "thnn_"; "th_"; "_foreach" ]
+let excluded_prefixes = [ "_thnn_"; "_th_"; "thnn_"; "th_"; "_foreach"; "_amp" ]
```

Comment:
This removes `internal_amp_update_scale`, which is needed for mixed-precision training :(.

Reply:
I'll do a PR.